src/main/scala/intellij/haskell/external/repl/ProjectStackReplProcess.scala

```diff
@@ -47,20 +48,24 @@ private[repl] class ProjectStackReplProcess(project: Project) extends StackReplP
   }
 
   private def load(psiFile: PsiFile, moduleName: Option[String]): Option[(StackReplOutput, Boolean)] = {
-    val filePath = getFilePath(psiFile)
-    val moduleNameAction = ApplicationManager.getApplication.runReadAction(new Computable[Option[String]] {
-      override def compute(): Option[String] = {
-        moduleName.orElse(HaskellPsiUtil.findModuleName(psiFile))
+    if (!HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
Because [HaskellConsoleView](https://github.com/zjhmale/intellij-haskell/blob/4735b70def2a41f82f471eb760466aedd4a8bb1b/src/main/scala/intellij/haskell/runconfig/console/HaskellConsoleView.scala) will be treated as a PsiFile, so when you start an REPL, stack will load it as a common Haskell file just like the picture showing below

<img width="626" alt="screen shot 2017-02-23 at 12 49 44 am" src="https://cloud.githubusercontent.com/assets/6234553/23222153/02bd3512-f962-11e6-95ef-f495dab4e7c8.png">

So we'd better skip this loading process for console view.